src/ImageSharp.Web/Resolvers/PhysicalFileSystemCacheResolver.cs

```diff
@@ -33,7 +32,7 @@ public PhysicalFileSystemCacheResolver(IFileInfo metaFileInfo, FormatUtilities f
         /// <inheritdoc/>
         public async Task<ImageCacheMetadata> GetMetaDataAsync()
         {
-            using Stream stream = this.metaFileInfo.CreateReadStream();
+            using Stream stream = this.metaFileInfo.OpenRead();
```

Comment:
`OpenRead` will allocate a buffer. We need to copy the implementation [here](https://source.dot.net/#Microsoft.Extensions.FileProviders.Physical/PhysicalFileInfo.cs,46).

Reply:
Done in https://github.com/SixLabors/ImageSharp.Web/pull/225/commits/bc0ac62da85458aa3c8d19b0f3b9ff48e7c01359 👍🏻